cloudformation.yaml

```
@@ -589,6 +600,36 @@ Resources:
         Stack: !Ref Stack
         App: !Ref App
 
+  FeastApplePubSubLambda:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: feast-apple-pubsub.handler
+      Runtime: nodejs20.x
+      CodeUri:
+        Bucket: !Ref DeployBucket
+        Key: !Sub ${Stack}/${Stage}/${App}-feast-apple-pubsub/feast-apple-pubsub.zip
+      FunctionName: !Sub ${App}-feastapplepubsub-${Stage}
+      Role: !GetAtt MobilePurchasesLambdasRole.Arn
+      Environment:
+        Variables:
+          Stage: !Ref Stage
+          Stack: !Ref Stack
+          App: !Ref App
+      Description: Records play store events
```

Tiny suggestion for the sake of being more precise

```suggestion
      Description: Records app store events for Feast app
```